app/src/app.vue

```diff
@@ -14,7 +14,13 @@
 			</template>
 		</v-info>
 
-		<router-view v-else-if="!hydrating" />
+		<template v-else-if="!hydrating">
+			<component :is="view" v-if="view">
+				<router-view />
+			</component>
+
+			<router-view v-else />
+		</template>
```

Comment:
This is temporary so I can keep clicking around the app to/from pages that aren't using the new setup yet